uvloop/handles/process.pyx

```diff
@@ -101,6 +101,7 @@ cdef class UVProcess(UVHandle):
             self._finish_init()
 
             os_close(self._errpipe_write)
+            self._errpipe_write = -1
```

Comment:
the write_pipe can be closed again sometimes:

https://github.com/MagicStack/uvloop/blob/3e71ddc3383a8e0546519117c8775323c19eb6d7/uvloop/handles/process.pyx#L118-L122

if another thread allocates a file descriptor in this time then instead of raising OSError here the fd will be closed incorrectly